include/tpm2-tss-engine.h

```diff
@@ -34,10 +34,15 @@
 #include <openssl/engine.h>
 #include <tss2/tss2_tpm2_types.h>
 
+typedef enum { blob, handle } KEY_TYPE ;
```

Comment:
Please namespace the enum members, preferably with the enum typename, i.e. KEY_TYPE_BLOB

Reply:
Okay can I use tpm_tools scripts in my test, in order to generate a persistent handle ?